src/main/sensors/gyroanalyse.c

```diff
@@ -337,23 +337,20 @@ static FAST_CODE_NOINLINE void gyroDataAnalyseUpdate(gyroAnalyseState_t *state,
         {
             // 7us
             switch (dynamicFilterType) {
-            case FILTER_PT1: {
-                const int cutoffFreq = state->centerFreq[state->updateAxis] * dynamicFilterWidthFactor;
-                const float gyroDt = gyro.targetLooptime * 1e-6f;
-                const float gain = pt1FilterGain(cutoffFreq, gyroDt);
-
-                pt1FilterUpdateCutoff(&dynFilter[state->updateAxis].pt1FilterState, gain);
-
-                break;
-            }
-            case FILTER_BIQUAD: {
-                // calculate cutoffFreq and notch Q, update notch filter
-                const float cutoffFreq = fmax(state->centerFreq[state->updateAxis] * dynamicFilterWidthFactor, dynamicNotchMinCutoffHz);
-                const float notchQ = filterGetNotchQ(state->centerFreq[state->updateAxis], cutoffFreq);
-                biquadFilterUpdate(&dynFilter[state->updateAxis].biquadFilterState, state->centerFreq[state->updateAxis], gyro.targetLooptime, notchQ, FILTER_NOTCH);
-
-            break;
-            }
+                case FILTER_PT1: {
```

Comment:
`case` labels shouldn't be indented. You had `break` with a wrong indent, now everything has a wrong indent

Reply:
<groan> OK will fix

Reply:
Hopefully is right now